### PR TITLE
Migrate most ad-hoc styling to the css

### PR DESCRIPTION
**Pull request by andralex**
A bunch of styling is currently hardcoded in the macros. This work migrates most styling into the css style file. That way, we can rely on css styling methods to improve the look and feel of the site without doing surgery on the macros.

### CODE REVIEW

**Comment by andralex** [comment]
Here are all styles now available. Some are not defined in the css file. I defined the minimum that preserves the current look and feel.

```
class="anchor"
class="answer-nojs"
class="blue"
class="bnf notranslate"
class="book"
class="btn"
class="bugsfixed"
class="button"
class="ccode notranslate"
class="comp"
class="compNo"
class="compYes"
class="console notranslate"
class="cppcode notranslate"
class="cppcode2"
class="d_code notranslate"
class="d_code"
class="d_code2"
class="d_comment"
class="d_inlinecode donthyphenate notranslate"
class="d_inlinecode"
class="d_keyword"
class="d_param"
class="d_string"
class="ddoccode notranslate"
class="donthyphenate"
class="download"
class="download_image"
class="download_paragraph"
class="footnote"
class="gname"
class="green"
class="hyphenate"
class="metacode"
class="navblock"
class="nobr"
class="question"
class="quickindex"
class="red"
class="scini notranslate"
class="tip"
class="tocheader"
class="twitter-timeline"
class="version"
```

File: bugstats.php.dd [hunk 1]
```diff
@@ -43,6 +43,6 @@ $(P <center>$(LINK2 https://issues.dlang.org/reports.cgi?product=D&datasets=NEW&
 Macros:
   TITLE=The D Bug Tracker
   PHP=<?php $0 ?>
-  BOOKTABLE = <center><table cellspacing=0 cellpadding=5 valign=top class=book><caption>$1</caption>$2</table></center>
+  BOOKTABLE = <center><table cellspacing=0 cellpadding=5 valign=top class="book"><caption>$1</caption>$2</table></center>
```
Comment (WebDrake) [inline bugstats.php.dd]: Shouldn't stuff like the cellspacing, cellpadding, etc. also be moved to CSS?  Or are you thinking of doing this in a 3-stage process, first introducing the CSS references, then implementing the actual CSS design, and finally stripping out last hard-coded style elements?

Comment (WebDrake) [inline bugstats.php.dd]: Ack, my bad, didn't notice at first this was documenation, not the actual macro definition.  But in any case, shouldn't it be updated to match the actual current definition?

File: dlang.org.ddoc [hunk 2]
```diff
@@ -50,30 +52,31 @@ _=
...
+DDOC_AUTHORS   = $(DDOCKEYVAL Authors, $0)
+DDOC_BUGS      = $(DDOCKEYVAL Bugs, $0)
+DDOC_COPYRIGHT = $(DDOCKEYVAL Copyright, $0)
+DDOC_DATE      = $(DDOCKEYVAL Date, $0)
+DDOC_DEPRECATED = $(DDOCKEYVAL Deprecated, $0)
+DDOC_EXAMPLES  = $(DDOCKEYVAL Examples, $0)
+DDOC_HISTORY   = $(DDOCKEYVAL History, $0)
+DDOC_LICENSE   = $(DDOCKEYVAL License, $0)
+DDOC_RETURNS   = $(DDOCKEYVAL Returns, $0)
+DDOC_SEE_ALSO  = $(DDOCKEYVAL SeeAlso, $0)
```
Comment (quickfur) [inline dlang.org.ddoc]: This change causes the "See Also:" heading to be squashed into "SeeAlso:" in the HTML output. For example, see: http://dlang.org/phobos/std_string#.translate

Comment (quickfur) [inline dlang.org.ddoc]: https://github.com/D-Programming-Language/dlang.org/pull/908

**Comment by andralex** [comment]
Thanks!

**Pull request merged**